react-challenge/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```

Comment:
Should there be a plain old `.env` here?

Reply:
I think the standard usage is that we check a .env with non-secret standard values into the repo that is overridden by specific environment files that are never checked in

Reply:
I don't think it matters cause there's not actually any .env setup needed for the code challenge